src/praisonai-bot/praisonai_bot/bots/_config_schema.py

```
@@ -869,6 +882,112 @@ def normalize_and_validate(self):
+                f"imported: {exc}"
+            ) from exc
+        try:
+            adapter_cls = getattr(module, class_name)
+        except AttributeError as exc:
+            raise ValueError(
+                f"Channel adapter class '{class_name}' not found in "
+                f"module '{module_path}'."
+            ) from exc
+        try:
+            from praisonaiagents.bots import BasePlatformAdapter
+        except Exception:  # pragma: no cover - core always present in-tree
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)